tensorflow/compiler/mlir/tosa/tests/tfl-to-tosa-pipeline.mlir

```diff
@@ -54,11 +53,14 @@ func @test_transpose_conv2d(%arg0: tensor<1x32x32x8xf32>, %cst_0: tensor<16x1x1x
+// CHECK-DAG:           %[[VAR_2:.*]] = "tosa.const"() {value = dense<{{.*}}> : tensor<16x2x2x8x!quant.uniform<i8<-127:127>:f32:0, {1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01}>>} : () -> tensor<16x2x2x8x!quant.uniform<i8<-127:127>:f32:0, {1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01}>>
+// CHECK-DAG:           %[[VAR_3:.*]] = "tosa.conv2d"(%[[VAR_0]], %[[VAR_2]], %[[VAR_1]]) {dilation = [1, 1], pad = [0, 1, 0, 1], quantization_info = {input_zp = 0 : i32, weight_zp = 0 : i32}, stride = [1, 1]} : (tensor<1x32x32x8x!quant.uniform<i8:f32, 0.015684768557548523>>, tensor<16x2x2x8x!quant.uniform<i8<-127:127>:f32:0, {1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01,1.000000e-01}>>, tensor<16x!quant.uniform<i32:f32:0, {2.000000e+00,2.000000e+00,1.000000e+00,1.000000e+00,1.000000e+00,2.000000e+00,2.400000e+00,1.700000e+00,2.300000e+00,2.400000e+00,2.400000e+00,2.300000e+00,2.100000e+00,2.400000e+00,2.100000e+00,2.400000e+00}>>) -> tensor<1x32x32x16x!quant.uniform<i32:f32:0, {2.000000e+00,2.000000e+00,1.000000e+00,1.000000e+00,1.000000e+00,2.000000e+00,2.400000e+00,1.700000e+00,2.300000e+00,2.400000e+00,2.400000e+00,2.300000e+00,2.100000e+00,2.400000e+00,2.100000e+00,2.400000e+00}>>
+// CHECK-DAG:           %[[VAR_4:.*]] = "tosa.rescale"(%[[VAR_3]]) {double_round = true, input_zp = 0 : i32, multiplier = [1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32, 1374257539 : i32], output_zp = 0 : i32, per_channel = true, scale32 = true, shift = [36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32, 36 : i32]} : (tensor<1x32x32x16x!quant.uniform<i32:f32:0, {2.000000e+00,2.000000e+00,1.000000e+00,1.000000e+00,1.000000e+00,2.000000e+00,2.400000e+00,1.700000e+00,2.300000e+00,2.400000e+00,2.400000e+00,2.300000e+00,2.100000e+00,2.400000e+00,2.100000e+00,2.400000e+00}>>) -> tensor<1x32x32x16x!quant.uniform<i8:f32, 0.078431375324726104>>
+// CHECK-DAG:           return %[[VAR_4]] : tensor<1x32x32x16x!quant.uniform<i8:f32, 0.078431375324726104>>
```

Comment:
This needs to  be a CHECK not a CHECK-DAG. This with no base CHECK this CHECK-DAG never occurs

Reply:
Done